com.meli.superheroes/src/model/Figura.java

```diff
@@ -0,0 +1,64 @@
+package model;
+
+
+
+public class Figura {
+
+    private String codigo;
+    private double precio;
+    private Dimension dimension;
+    private SuperHeroe superHeroe;
+
+    public Figura(String codigo, int precio, Dimension dimension, SuperHeroe superHeroe) {
+        this.codigo = codigo;
+        this.precio = precio;
+        this.dimension = dimension;
+        this.superHeroe = superHeroe;
+    }
+    public void subirPrecio (double valorAdicional){
+         double valorSubido =getPrecio() + valorAdicional;
+         setPrecio(valorSubido);
```

Comment:
Ok, tambien podes agregar el calculo en 1 sola linea.
como buena practica es mas correcto llamar a los metodos dentro de una clase con la palabra this, seria:
this.setPrecio(this.getPrecio() + valorAdicional);